db/migrate/20210104230650_create_profile.rb

```diff
@@ -0,0 +1,12 @@
+class CreateProfile < ActiveRecord::Migration[5.2]
+  def change
+    create_table :profiles do |t|
+      t.string :name
+      t.integer :age
+      t.float :height
+      t.float :weight
+      t.float :goal_weight
+      t.references :user, foreign_key: true
```

Comment:
Look into removing this foreign key. Probably dont need since im not using active record association here.

Reply:
Not going to mess with the schema right now.